### PR TITLE
added links to blogs except 1st one

### PR DESCRIPTION
**Pull request by SanikaBhalerao1345**
### Pull Request Checklist

- [X] I have added screenshots and videos to show before and after the working of my code.
- [X] I have ensured that the screen size is set to 100% while making the video.
- [X] I have synced the latest fork with my local repository and resolved any conflicts.
- [X] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [X] I understand that if any the above conditions are not met , my PR will not be MERGED .
I have linked roaming routes to blog.html and the rest to other blogs.
issue number: #1702

### CODE REVIEW

**Comment by SanikaBhalerao1345** [comment]
@ANSHIKA-26 pls check my pr

**Comment by ANSHIKA-26** [comment]
@SanikaBhalerao1345 kindly remove all links to external blogs

File: index.html [hunk 1]
```diff
@@ -816,7 +816,7 @@ <h2>Forgot Password</h2>
                             <!-- card for changingg -->
                             <div class="card">
                                 <div class="overflow-img">
-                                    <a href="#">
+                                    <a href="https://www.gameopedia.com/blog/ ">
                                         <img style="height: 250px;width:300px" src="./assets/img2.webp"
```
Comment (ANSHIKA-26) [inline index.html:820]: remove these links

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by SanikaBhalerao1345** [comment]
@ANSHIKA-26 pls check i have raised a new pr

**Pull request closed without merging**